src/main/org/apache/tools/ant/UnknownElement.java

```diff
@@ -151,11 +151,7 @@ public RuntimeConfigurable getWrapper() {
     }
 
     /**
-     * Creates the real object instance and child elements, then configures
-     * the attributes and text of the real object. This unknown element
-     * is then replaced with the real object in the containing target's list
-     * of children.
-     *
+     * Configure the given object from this UnknownElement
```

Comment:
This replaces a somewhat lengthy detailed description of what happens by a less clear one-liner. Why is this better?